README.md

```diff
@@ -37,26 +37,24 @@
 
 ## Nakadi Event Broker
 
-The goal of the Nakadi project (ნაკადი means "stream" in Georgian) is 
-to build an event broker infrastructure to:
+The goal of Nakadi (ნაკადი means "stream" in Georgian) is to provide a technology neutral and efficient event broker infrastructure that allows convenient development of event-driven applications and asynchronous microservices. 
 
-*  Allow event exchange via a RESTful [API](/api/nakadi-event-bus-api.yaml)
-*  Enable convenient development of event-driven applications
-*  Securely and efficiently publish and consume events
+- **HTTP API with OAuth**. Publish and consume events via a [RESTful API](/api/nakadi-event-bus-api.yaml) making integration with any technology stack easy and allowing microservices teams to maintain service boundaries. Access to the API can be managed and secured using OAuth scopes.
 
-Some additional requirements for the project are to provide:
+- **Standard Event Types**. Event types for business processes and data changes can be defined based on common primitives for identity, timestamps, event types, and causality. 
 
-* Event ordering guarantees
-* Low latency (near real-time) event processing
-* Scalable and highly available architecture
-* Compatability with the [STUPS project](https://stups.io/)
+- **Schema Registry and Event Validation**. Event types are created and managed via a Registry API. Events can be extended based on schemas and validated before they are made available to consumers.
 
-Additional topics, that we plan to cover are:
+- **Low Latency Consumption**. Consumers can be pushed to via a streaming HTTP connection, allowing near real-time event processing. The consumer connection has keepalive controls and support for managing stream offsets.
+
+The project also provides compatability with the [STUPS project](https://stups.io/). Additional features that we plan to cover in the future are:
+
+* Discoverability of the resource structures flowing into the broker
 
-* Discoverability of the resource structures flowing into the event broker
-* A schema registry, to collect schema information for easy lookup by developers
 * A managed API that allows consumers to subscribe and have stream offsets stored by the server
 
+* Filtering of events for subscribing consumers
```

Comment:
We can (I didn't want to anything extra to the roadmap without consensus, but afaik filtering is something the team wants to look at)